spec/integration/support/serve_examples.rb

```diff
@@ -31,15 +32,17 @@
         group: machinery_config[:group]
       )
 
-      cmd = "#{machinery_command} serve jeos --port 5000"
+      cmd = "#{machinery_command} serve opensuse131 --port 5000"
       Thread.new do
         @machinery.run_command(cmd)
       end
 
       # Test basic HTML
       wait_time = 0
+        require 'byebug'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.